splainer-search.js

```diff
@@ -2616,7 +2616,11 @@ angular.module('o19s.splainer-search')
               errorMsg +=  '\n';
               errorMsg +=  'Enable CORS in elasticsearch.yml:\n';
               errorMsg += '\n';
+<<<<<<< HEAD
```

Comment:
@softwaredoug there's some merge conflict artifacts in here

Reply:
Pushed up a fix